director/projects/views/project_views.py

```diff
@@ -281,15 +278,15 @@ def post(  # type: ignore
         project_name: str,
         path: typing.Optional[str] = None,
     ) -> HttpResponse:
-        project = self.get_project(request.user, account_name, project_name)
+        self.perform_project_fetch(request.user, account_name, project_name)
```

Comment:
Not using `get_project` here because the POST requires EDIT permissions but the get requires just VIEW.